internal/rollout/rollout.go

```diff
@@ -104,28 +104,33 @@ func (r *Rollout) Rollout() (bool, error) {
 		"region":  r.region,
 	})
 
-	svc, err := r.UpdateService(r.service)
+	_, trafficChanged, err := r.UpdateService(r.service)
```

Comment:
I'm not using the returned service here, but it's useful for testing since we can test that we got the expected changes, not only that the traffic has changed.